gatsby-browser.js

```diff
@@ -1 +1,7 @@
 export { wrapRootElement } from './src/apollo/wrap-root-element'
+
+export const onInitialClientRender = () => {
+  const addThisScript = document.createElement('script')
+  addThisScript.src = `//s7.addthis.com/js/300/addthis_widget.js#pubid=${process.env.GATSBY_ADDTHIS_PUB_ID}`
```

Comment:
@duranmla let's add a check here to only render this if `GATSBY_ADDTHIS_PUB_ID` variable is present

Reply:
It indeed only renders when the env variable is rendered. But we can avoid to create the script tag when the variable is not there.

Reply:
Addressed here 430f4ec